README.md

```diff
@@ -27,7 +39,7 @@ To generate a configuation file template, run the command below in your terminal
 cargo run -- print-config
 ```
 
-Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed cellar contract.
+Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed Cellar contract.
 
 You can create keys or import keys. To create keys, run the command below:
```

Comment:
It would be good to add an example of how to import keys as well.

Reply:
fixed